CATALOG.md

```diff
@@ -0,0 +1,42 @@
+# Build Commands Library
+
+Below you'll find a list of build commands contributed from the nativefier community.
+
+If you'd like to add to this catalog, please keep the following in mind:
+
+Library commands should be the strict necessary to make an app work. So, for example,
+
+- Yes to have `--widevine` in Udemy. Yes to `--internal-urls` and `--browserwindow-options` for Outlook Web. Yes to a name and icon for both...
+- ... but let's not have all the other flags that are a matter of personal preference (e.g. `--disable-dev-tools` or `--disk-cache-size`).
+
+* * *
+
+## Outlook Web
+
+### Windows Command
+```
+nativefier https://outlook.office.com/mail 
+--name OutlookWA 
```

Comment:
I think for these I'd leave `--name` off and either let it be inferred, or allow the user to decide, since this won't change the app behavior in any way.

Reply:
Makes sense... I will remove!!